erpnext/accounts/doctype/payment_request/payment_request.py

```
@@ -1007,3 +1013,59 @@ def get_irequests_of_payment_request(doc: str | None = None) -> list:
 			},
 		)
 	return res
+
+
+def set_payment_references(payment_request, ref_doc, args):
```

Ruff (0.14.7)

Unused function argument: `args`

(ARG001)